fieldservice_size/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+The module allows you to establish sizes for your field service orders.
+Multiple sizes can be configured and established for a location. Order
+size will be updated based on its location and order type.
```

Comment:
When reading this I don't get what can be  a "size" of FSM Order.
Can you add more explanation about what is it / when we need it ?

Reply:
@hparfr I updated the readme to help make the use of sizes more clear.

For me, my specific use is as follows:
We apply fertilizer to a lawn.  The lawn size is measured in square feet units.  Knowing the size of the lawn helps to estimate how many orders to plan for a worker in a day and how many units of fertilizer need to be used at each location.

Thank you for the review and let me know if I can further update the readme